### PR TITLE
Add SSO instructions

### PR DESCRIPTION
**Pull request by mayankshouche**

### CODE REVIEW

File: settings/authentication.mdx
Comment (joshcolts18) [inline settings/authentication.mdx:1]: @mayankshouche Would specify in the note that the functionality is available on our enterprise plan and link to our Calendly: https://calendly.com/mintlify/demo-your-docs

Comment (mayankshouche) [reply settings/authentication.mdx:1]: just added! lmk if that works

File: settings/authentication.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: 'Single sign-on'
```
Comment (hahnbeelee) [inline settings/authentication.mdx]: Can we capitalize and add SSO for Cmd+F-ability?
```suggestion
title: 'Single Sign-On (SSO)'
```

Comment (mayankshouche) [reply settings/authentication.mdx]: updated!

**Review by hahnbeelee — APPROVED** [review]
LGTM

**Pull request merged**